doc/configuration-v2_1-experimental.md

```diff
@@ -87,11 +87,21 @@ The Ignition configuration is a JSON document conforming to the following specif
     * **name** (string): the name of the file. This must be suffixed with a valid unit type (e.g. "00-eth0.network").
     * **_contents_** (string): the contents of the networkd file.
 * **_passwd_** (object): describes the desired additions to the passwd database.
-  * **_users_** (list of objects): the list of accounts to be added.
+  * **_users_** (list of objects): the list of accounts that should exist.
     * **name** (string): the username for the account.
     * **_passwordHash_** (string): the encrypted password for the account.
     * **_sshAuthorizedKeys_** (list of strings): a list of SSH keys to be added to the user's authorized_keys.
-    * **_create_** (object): contains the set of options to be used when creating the user. A non-null entry indicates that the user account shall be created.
+    * **_uid_** (integer): the user ID of the account.
+    * **_gecos_** (string): the GECOS field of the account.
+    * **_homeDir_** (string): the home directory of the account.
+    * **_noCreateHome_** (boolean): whether or not to create the user's home directory. This only has an effect if the account doesn't exist yet.
+    * **_primaryGroup_** (string): the name or ID of the primary group of the account.
```

Comment:
These docs are still out of date. Regarding primaryGroup, since the groups option only accepts strings, I would just make this do the same.